### ISSUE TITLE
Allow reflect-metadata (NodeVM)

### ISSUE THREAD

**Issue by TheAifam5** [patriksimek/vm2#147]
Error log:
```
[ExceptionHandler] 'get' on proxy: property 'prototype' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value
(expected '[object Object]' but got '[object Object]')
TypeError: 'get' on proxy: property 'prototype' is a read-only and non-configurable data property on the proxy target but the proxy did not return its actual value (expected '[object Object]' but got '[object Object]')
    at OrdinaryGetPrototypeOf (node_modules\reflect-metadata\Reflect.ts:1570:33)
    at OrdinaryGetMetadata (node_modules\reflect-metadata\Reflect.ts:1266:28)
    at Object.getMetadata (node_modules\reflect-metadata\Reflect.ts:996:20)
    at ModuleTokenFactory.reflectScope (node_modules\@nestjs\core\injector\module-token-factory.js:32:31)
    at ModuleTokenFactory.create (node_modules\@nestjs\core\injector\module-token-factory.js:7:37)
    at ModuleCompiler.compile (node_modules\@nestjs\core\injector\compiler.js:19:47)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by jpsfs** [comment]
@TheAifam5 any news on this?
I'm also facing difficulties using "reflect-metadata".

**Comment by XmiliaH** [comment]
This should be fixed in v3.9.1
